### PR TITLE
en: fix deploy-tidb-binlog.md

### PR DESCRIPTION
**Pull request by Joyinqin**
### What is changed, added or deleted? (Required)
- fix helm 2/3 command
- add attention block in pump nodes remove section

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  https://github.com/pingcap/docs-tidb-operator/pull/807
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: en/deploy-tidb-binlog.md [hunk 1]
```diff
@@ -396,6 +406,10 @@ The steps are as follows:
 
 ### Remove Pump nodes completely
 
+> **Note:**
+>
+> Before performing the following steps, you need to have at least one Pump node in the cluster. If the Pump nodes have been scaled in to `0` at this time, you need to scale out them at least to `1` before you remove them. To scale out the Pump to `1`, execute `kubectl edit tc ${tidb-cluster} -n ${namespace}` and modify the `spec.pump.replicas` to `1`.
```
Comment (ran-huang) [inline en/deploy-tidb-binlog.md:411]:
```suggestion
> - Before performing the following steps, you need to have at least one Pump node in the cluster. If the Pump cluster has been scaled in to `0` node at this time, you need to scale out the cluster at least to `1` node before you perform the removing operation in this section.
> - To scale out the Pump to `1`, execute `kubectl edit tc ${tidb-cluster} -n ${namespace}` and modify the `spec.pump.replicas` to `1`.
```

**Comment by ran-huang** [comment]
LGTM

File: en/deploy-tidb-binlog.md [hunk 2]
```diff
@@ -396,6 +406,11 @@ The steps are as follows:
 
 ### Remove Pump nodes completely
 
+> **Note:**
+>
+> - Before performing the following steps, you need to have at least one Pump node in the cluster. If the Pump cluster has been scaled in to `0` node at this time, you need to scale out the cluster at least to `1` node before you perform the removing operation in this section.
```
Comment (dragonly) [inline en/deploy-tidb-binlog.md:411]: I think "Pump cluster" is a bit strange, maybe we can just call it "Pump" and refer to the scale-in as "scale the Pump nodes to zero"?

File: en/deploy-tidb-binlog.md [hunk 3]
```diff
@@ -408,7 +408,7 @@ The steps are as follows:
 
 > **Note:**
 >
-> - Before performing the following steps, you need to have at least one Pump node in the cluster. If the Pump cluster has been scaled in to `0` node at this time, you need to scale out the cluster at least to `1` node before you perform the removing operation in this section.
+> - Before performing the following steps, you need to have at least one Pump node in the cluster. If the Pump has scaled the Pump nodes to `0` node at this time, you need to scale out the Pump at least to `1` node before you perform the removing operation in this section.
```
Comment (ran-huang) [inline en/deploy-tidb-binlog.md:411]:
```suggestion
> - Before performing the following steps, you need to have at least one Pump node in the cluster. If you have scaled in Pump nodes to `0`, you need to scale out Pump at least to `1` node before you perform the removing operation in this section.
```

**Comment by ran-huang** [comment]
@dragonly PTAL again. Thanks!

**Comment by dragonly** [comment]
LGTM

**Pull request merged**